src/components/VisualizationScreen.tsx

```diff
@@ -10,7 +10,7 @@ const bifrostWidgetCss = css`
   // Element-based styles
   //===========================================================
   display: grid;
-  grid-template-columns: 2fr 1fr;
+  grid-template-columns: 500px 400px;
```

Comment:
If the size is fixed, then there could be a space to the right. Shouldn't it proportionally grow depending on the window size? This may be solved if we figure out how to set the size of bifrost widget fixed. No need to change if this is temporary.